src/main/java/com/adyen/httpclient/HttpURLConnectionClient.java

```diff
@@ -56,12 +56,16 @@ public String request(String requestUrl, String requestBody, Config config) thro
     public String request(String requestUrl, String requestBody, Config config, boolean isApiKeyRequired) throws IOException, HTTPClientException {
         HttpURLConnection httpConnection = createRequest(requestUrl, config.getApplicationName());
         String apiKey = config.getApiKey();
+        int connectionTimeoutMillis = config.getConnectionTimeoutMillis();
         // Use Api key if required or if provided
         if (isApiKeyRequired || (apiKey != null && !apiKey.isEmpty())) {
             setApiKey(httpConnection, apiKey);
         } else {
             setBasicAuthentication(httpConnection, config.getUsername(), config.getPassword());
         }
+        if (connectionTimeoutMillis > 0) {
+            setConnectionTimeout(httpConnection, connectionTimeoutMillis);
```

Comment:
since the arguments and number of lines are the same, why not just use the httpConnection.setConnectTimeout(connectionTimeout) instead of wrapping it within a helper function?

Reply:
It will give more flexibility to set it directly for HttpURLConnectionClient object even after object is already created and later connectiontimeout needs to be set. If we directly use httpConnection.setConnectionTimout(), we will loose this flexibility.